sw/device/silicon_creator/rom/rom.c

```diff
@@ -401,25 +401,21 @@ static rom_error_t rom_verify(const manifest_t *manifest,
    * We swap the order of signature verifications randomly.
    */
   *flash_exec = 0;
-  if (rnd_uint32() < 0x80000000) {
-    HARDENED_RETURN_IF_ERROR(sigverify_ecdsa_p256_verify(
-        &manifest->ecdsa_signature, ecdsa_key, &rev_digest, flash_exec));
-
-    return sigverify_spx_verify(
-        spx_signature, spx_key, spx_config, lc_state,
-        &usage_constraints_from_hw, sizeof(usage_constraints_from_hw),
-        anti_rollback, anti_rollback_len, digest_region.start,
-        digest_region.length, &fwd_digest, flash_exec);
-  } else {
-    HARDENED_RETURN_IF_ERROR(sigverify_spx_verify(
-        spx_signature, spx_key, spx_config, lc_state,
-        &usage_constraints_from_hw, sizeof(usage_constraints_from_hw),
-        anti_rollback, anti_rollback_len, digest_region.start,
-        digest_region.length, &fwd_digest, flash_exec));
-
-    return sigverify_ecdsa_p256_verify(&manifest->ecdsa_signature, ecdsa_key,
-                                       &rev_digest, flash_exec);
-  }
+
+  HARDENED_RETURN_IF_ERROR(sigverify_ecdsa_p256_start(
```

Comment:
This change is a good candidate for master, but I think we can leave from `earlgrey_1.0.0` for now.

Reply:
ack.  Added to my list.